### PR TITLE
Added some missing notes to common_structs.h

### PR DESCRIPTION
**Pull request by eldexterr**
Added missing notes from BattleStatus(0x172) & (0x174)

### CODE REVIEW

**Comment by BowserSlug** [comment]
✅ This PR fixes 4178 warnings!

File: include/common_structs.h [hunk 1]
```diff
@@ -714,7 +714,8 @@ typedef struct BattleStatus {
     /* 0x140 */ s16 enemyIDs[24];
     /* 0x170 */ char unk_170;
     /* 0x171 */ s8 numEnemyActors;
-    /* 0x172 */ char unk_172[6];
+    /* 0x172 */ s16 currentTurnActorID;
+    /* 0x174 */ s32 Actor* currentTurnActor;
```
Comment (ethteck) [inline include/common_structs.h:718]: You have two types for this field declaration: `s32` and `Actor*` - only one should be there

**Review by ethteck — CHANGES_REQUESTED** [review]
Looks good other than my comment! Where are these changes coming from, btw?

**Review by nanaian — COMMENTED** [review]
> This PR fixes 4286 warnings!

whoa.

I'll defer to ethan on this one review-wise

File: include/common_structs.h (hunk 1 above)
Comment (eldexterr) [reply include/common_structs.h:718]: Hmm is the pointer of data to the enemy attacking so I guess should be like this?
`struct Actor* currentTurnActor`

I just noticed that by looking at the RAM, but the code that sets those values is at ~Func:btl_state_update_next_enemy (802472AC, 175B8C), also looks like I made some mistakes and is actually "currentTurnEnemyID" It doesn't apply for Player or Partner, only for enemies.

So I think a better approach should be
"currentTurnEnemyID" & "currentTurnEnemy"

Comment (ethteck) [reply include/common_structs.h:718]: Ok, sounds good! And yeah I think that type makes more sense.

**Review by ethteck — APPROVED** [review]

**Pull request merged**